namerd/README.md

```diff
@@ -33,7 +33,7 @@ a long-poll thrift interface.
 Namerd can be run locally with the commands
 
 ```
-./sbt namerd-examples/basic:run
+./sbt namerd/examples/basic:run
```

Comment:
Hey @AsCat -- thanks for the PR! The previous version of this command works on my machine:

```
$ ./sbt namerd-examples/basic:run
...
I 0302 11:13:15.570 PST THREAD83: serving http on /127.0.0.1:9991
I 0302 11:13:16.048 PST THREAD83: Tracer: com.twitter.finagle.zipkin.thrift.ScribeZipkinTracer
I 0302 11:13:16.111 PST THREAD83: serving io.l5d.thriftNameInterpreter interface on /127.0.0.1:4100
I 0302 11:13:16.117 PST THREAD83: serving io.l5d.httpController interface on /127.0.0.1:4180
```

Whereas the updated version does not:

```
$ ./sbt namerd/examples/basic:run
...
[error] Expected configuration
[error] Expected ':' (if selecting a configuration)
[error] Expected key
[error] Not a valid key: examples (similar: compilers, name, extra-loggers)
[error] namerd/examples/basic:run
```

The issue is that "namerd-examples" is an sbt project name, and projects names can't contain "/" characters, so they don't map directly to filesystem paths.  In fact, our sbt configuration has a [helper method](https://github.com/linkerd/linkerd/blob/master/project/Base.scala#L195) for converting paths to project names, by replacing "/" characters with "-" characters.